### PR TITLE
Tickets/sitcom 798

### PR DESCRIPTION
**Pull request by nsevilla**
After some merge conflict resolution and a bit of commit squashing, I am making this PR for the complete creation and development of this notebook to associated with SITCOM-798.

### CODE REVIEW

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,655 @@
+{
```
Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>1. Pleae, change the notebook title to something more meaningful like: "SITCOM-798_settling_time_after_slewing.ipynb"</p><p> </p><p>2. Break the following phrase down into simpler and more concise sentences. I found it a litte bit hard to follow.</p><p> </p><p>"This notebook will return a FAIL/PASS after the last cell if the RMS of the displacement values with respect to a long term value of positions and rotations measured by the IMS at M1M3 exceed/not exceed the IMS repeatability error in a certain required time range after the slew stop"</p><p> </p><p>Here is my suggestion:</p><p> </p><p>"This notebook evaluates the RMS of displacement values against long-term position and rotation measurements taken by the IMS at M1M3. It returns a PASS if these values are within the IMS repeatability error and a FAIL if they exceed it. The evaluation occurs within a specific time range after the slew stops."</p><p> </p><p>3. I think that is better if you say explicitly what is the time range required. I believe that all that information will be in the technote, but it is also a good a idea to give a bit more of context here in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701456'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='0'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>Please, load the "lab_black" extension here ("%load_ext lab_black) to make sure each cell comply with black formatting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701461'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='1'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>I believe you did some testing ultil you finally picked that specific slew (i_slew = 450). What about giving some context here on how did you get it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701464'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #21.        ax.axvline(pd.to_datetime(reference_t), lw="0.5", c="k")
</pre><p> </p><p>I don't get what the black line is indicating here. Is it indicating the azimuth mount behavior after a slew? Was this a azimuth slew from 0 to ~3.5, while the mount should remain stationary in elevation (but we actually see some variation?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701468'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='17'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>"This <s>is </s>step..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701470'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='22'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>I think you should add some comments on what you are seeing in this graph to make the reader follow the same logic you are thinking...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701472'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>Those plots are really nice. My only suggestion here would be to add a summary guiding the reader to the conclusion, based on the hypothesis raised above of what a "PASSED" test means. I think this would close the loop on what you want to show, also making the reader's life easier. Please, do the same for the "FAILED" example below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701475'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='26'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #7.                        rmsReq = req_rms, 
</pre><p> </p><p>When running this cell I get the following error (same happening in the cells below(: </p><pre class="ql-syntax" spellcheck="false">NameError: name 'req_rms' is not defined
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701477'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='28'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #7.                        rmsReq = req_rms, 
</pre><p> </p><pre class="ql-syntax" spellcheck="false">NameError: name 'req_rms' is not defined
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701480'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='29'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <pre class="ql-syntax" spellcheck="false">Line #7.                        imsColumn = 'yPosition', 
</pre><p> </p><pre class="ql-syntax" spellcheck="false">NameError: name 'req_rms' is not defined
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701483'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='30'/>

Comment (dsanmartim) [inline notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: <p>My final comment is more of a question. You have the function to determine the status of a single test, right? So, what now? Do you have to create a statistic for thefinal test we are running now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/#comment-1400701484'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

**Review by dsanmartim — COMMENTED** [review]
Hi Nacho,

I think it is almost ready, but I asked you a couple of questions and requested a few minor changes. Tks!

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb (hunk 1 above)
Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Fixed

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Added

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: It is actually a more or less random choice for that day, for illustration purposes. I will clarify this.

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: This is an auxiliary function mainly to illustrate the mount motion together with the IMS position measurements. The black line shows the slew.end according to the EventMaker. Added this info in a legend in this auxiliary plot.

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Done

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Removed this cell (deprecated, it is a copy of the previous one with a different column but with an obsolete name). Actually only the first is necessary, and then you would change the IMS column name of interest.

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Removed this cell (deprecated, it is a copy of the previous one with a different column but with an obsolete name). Actually only the first is necessary, and then you would change the IMS column name of interest.

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Removed this cell (deprecated, it is a copy of the previous one with a different column but with an obsolete name). Actually only the first is necessary, and then you would change the IMS column name of interest.

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Indeed, now I am in the process of generalizing this for set of tests throughout the night. And at the same time, settle on the details of the PASS/FAlL criteria, which we have not converged on 100%.

Comment (nsevilla) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: I added a clarifying sentence for each outcome. Let me know what you think.

**Comment by nsevilla** [comment]
> Hi Nacho,
> 
> I think it is almost ready, but I asked you a couple of questions and requested a few minor changes. Tks!

Thanks @dsanmartim I replied to all your comments and closed those comments with minor changes, others I left open. I proceed to reopen the PR.

File: notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb (hunk 1 above)
Comment (dsanmartim) [reply notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb]: Perfect!

**Review by dsanmartim — APPROVED** [review]
